contracts/roles/OwnerManager.sol

```diff
@@ -219,12 +262,12 @@ contract OwnerManager is OwnerRoles {
      *  OwnerManager has to be set as owner on the Claim Topics Registry smart contract to process this function
      *  See {IClaimTopicsRegistry-addClaimTopic}.
      *  Requires that `_onchainID` is set as ClaimRegistryManager on the OwnerManager contract
-     *  Requires that msg.sender is a MANAGEMENT KEY on `_onchainID`
+     *  Requires that msg.sender is an ACTION KEY on `_onchainID`
      *  @param _onchainID the _onchainID contract of the caller, e.g. "i call this function and i am Bob"
      */
     function callAddClaimTopic(uint256 _claimTopic, IIdentity _onchainID) external {
         require(
-            isClaimRegistryManager(address(_onchainID)) && _onchainID.keyHasPurpose(keccak256(abi.encode(msg.sender)), 1),
+            isClaimRegistryManager(address(_onchainID)) && _onchainID.keyHasPurpose(keccak256(abi.encode(msg.sender)), 2),
```

Comment:
Why putting 2 instead of true or false?

Reply:
i don't understand your comment, the `2` is the key type. 
Key type 1 is MANAGEMENT key 
key type 2 is ACTION key
key type 3 is CLAIM signing key
etc.

Reply:
function isClaimRegistryManager(address _owner) public view returns (bool)
Returns a boolean

Reply:
I might have misunderstood something

Reply:
https://github.com/TokenySolutions/T-REX/blob/04d1093a41f1e208d6f6c430f4538e0117612c9a/test/ownerManager.test.js#L172

That's how it is used in the tests

Reply:
https://github.com/onchain-id/solidity/blob/master/contracts/interface/IERC734.sol#L103

 keyHasPurpose returns a bool as well

Reply:
Oh, my bad it's all good! ✌️ I just saw the parenthesis 😅